mmv1/products/workstations/WorkstationConfig.yaml

```
@@ -428,6 +428,20 @@ properties:
         description: |
           The service account to use with the specified KMS key.
         required: true
+  - !ruby/object:Api::Type::Array
+    name: 'readinessChecks'
+    description: |
+      Readiness checks to be performed on a workstation.
+    item_type: !ruby/object:Api::Type::NestedObject
+      properties:
+        - !ruby/object:Api::Type::String
+          name: 'path'
+          description: |
+            Path to which the request should be sent.
+        - !ruby/object:Api::Type::String
```

```suggestion
        - !ruby/object:Api::Type::Integer
```
The provider crashes with error `readiness_checks.0.port: '' expected type 'string', got unconvertible type 'float64', value: '80'`

---

Woops, thanks! Fixed.